.codespell.exclude

```diff
@@ -30,6 +30,7 @@
 		"rasi it maka makpes"                   # fish en papillote
 		"Far Upugf Enrel Ugt"                 # Nam Myoho Renge Kyo (Buddhist religious chant)
 		"Far Uhudf Elrep Uds"                  # Nam Myoho Renge Kyo (Buddhist religious chant)
+		"Capitalise"		# (Miswritten)
```

Comment:
Ah. You need to put the entire line here in which the word appears.

Reply:
_Breathing intensifies harder._